index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script>
+      let question;
+
+      function tasuku(work) {
+        console.log("========================");
+        console.log("現在持っているタスク一覧");
+        console.log("========================");
+        work.forEach((i, index) => {
+          console.log(`${index}: ${i}`);
+        });
+      }
+
+      var work = ["掃除", "買い物", "散歩"];
```

Comment:
`var` は昨今ではほとんど使われません。
今回は再代入はしないので `const` がよいと思います。

```suggestion
      const work = ["掃除", "買い物", "散歩"];
```